src/robot_interface/models/mission/step.py

```diff
@@ -54,6 +54,9 @@ def robot_class_to_pretty_string(obj: Step) -> str:
         class_name: str = type(self).__name__
         return class_name + robot_class_to_pretty_string(self)
 
+    def to_dict(self) -> dict:
+        return {"id": self.id, "type": self.__class__.__name__}
```

Comment:
Should this dict also contain the status of the step?

Reply:
At the beginning of the mission all steps and tasks are set as `NotStarted`, so there is no need to send this. A possibility is to include tasks that we were unable to plan (not able to find predefined poses/coordinates from STID, unsupported inspection types).

Reply:
So this is only meant to be called in the beginning of the program?
Might be worth renaming it to reflect that.